styles/components/Search.module.scss

```diff
@@ -3,6 +3,11 @@
 
 .filterTags {
   margin-bottom: var(--nypl-space-xs);
+
```

Comment:
This is a DS styling override, maybe we should see whether this should be implemented there?